go.mod

```diff
@@ -2,6 +2,7 @@ module github.com/Azure/kubelogin
 
 // NOTE: kubelogin follows the same support policy as Go, which supports the last two major versions.
 go 1.21
+toolchain go1.22.3
```

Comment:
@bcho is it expected? should we change L4 as well?

Reply:
umm i think it's unexpected, but it won't break thing according to this doc: https://go.dev/blog/toolchain#:~:text=There%20is%20also%20a%20new%20toolchain%20line%20in%20go.mod%20that%20specifies%20the%20minimum%20Go%20toolchain%20to%20use%20when%20working%20in%20a%20particular%20module.%20In%20contrast%20to%20the%20go%20line%2C%20toolchain%20does%20not%20impose%20a%20requirement%20on%20other%20modules.%20For%20example%2C%20a%20go.mod%20might%20say%3A

Reply:
but looks like the build is failing already, i guess dependabot is having issues with these toolchain config

Reply:
yeah i think it's an implementation issue in the dependabot side, after using the toolchain, the go version should be set with patch version in the go directive. I think we probably need a separate PR to update the go.mod settings first.

Reply:
https://github.com/dependabot/dependabot-core/pull/8044 similar issue

Reply:
it is still wrong

Reply:
yep i tried the `go mod tidy` from local, it will add an empty line in between...